src/pages/docs/en/quickstart-guides/python.md

```diff
@@ -19,4 +19,29 @@ micros:
   - name: python-app
     src: ./src/python
     engine: python3.9
-```
\ No newline at end of file
+```
+
+## Setting up local development
```

Comment:
I think we should also have this for the other quickstarts, at least for Node and Custom though.